appservice/src/SiteWrapper.ts

```diff
@@ -537,7 +547,9 @@ export class SiteWrapper {
             await new Promise((resolve: () => void): void => { setTimeout(resolve, pollingInterval); });
             deployment = await kuduClient.deployment.getResult(deploymentId);
         }
-
+        if (deployment.id === previousDeploymentId) {
+            throw new Error(localize('unsuccessfulDeployment', 'Deployment was unsuccessful and fell back to last active deployment.'));
```

Comment:
How about something like:

  The deployment was unsuccessful and fell back to the last active deployment. If this is a local git deployment, it could also mean this repo has already been deployed.

Reply:
The deployment didn't actually fall back to the last deployment, though. If you navigate to the web app's url, you will see the broken app that you just deployed (you will _not_ see the last successful deployment).

The only thing that "fell back" is the deployment metadata reported by kudu when you query for 'latest'. But that's just an internal implementation detail to kudu and the user doesn't care about it.

In other words, I don't think we should mention "falling back" at all